src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyle.cs

```diff
@@ -274,25 +313,32 @@ public Size2D MaximumSize
             set => SetValue(ClippingModeProperty, value);
         }
 
-        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <summary>
+        /// Sets the size of a view for the width, the height and the depth.<br />
+        /// The views default depth is the minimum of width and height.<br />
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
         public Size Size
         {
             get => (Size)GetValue(SizeProperty);
             set => SetValue(SizeProperty, value);
         }
 
-        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <summary>
+        /// Gets or sets the Margin for use in layout.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
         public Extents Margin
         {
             get => (Extents)GetValue(MarginProperty) ?? (margin = new Extents());
             set => SetValue(MarginProperty, value);
         }
 
-        /// <summary> View BackgroundColor </summary>
-        /// This will be public opened in tizen_5.5 after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <summary>
+        /// Gets or sets the color of the background of view.
+        /// The mutually exclusive with "backgroundImage".
```

Comment:
what happens if both are set? is it a fatal error or one is ignored? which one? Does setting it overwrites backgroundImage?

Reply:
Setting it overwrites backgroundImage. I added more description. Please check.